openapi-annotations/build.gradle

```diff
@@ -1,7 +1,14 @@
 plugins {
-    id 'io.micronaut.build.internal.openapi-module'
+    id 'io.micronaut.build.internal.openapi-simple-module'
 }
 
 dependencies {
     api(libs.managed.swagger.annotations)
 }
+
+configurations.configureEach {
+
+    exclude group: "io.micronaut", module: "micronaut-inject-java"
```

Comment:
FYI I belive this syntax is deprecated in Gradle

Reply:
really? hm... what is correct syntax? I didn't find any information about it

Reply:
Not sure where I saw it, I think the prefered syntax is the full string one: "io.micronaut:micronaut-inject-java"

Reply:
@dstepanov I didn't find any information about deprecation of this syntax. Also I didn't find how to do it another way.

Also I moved all scripts to kotlin DSL here: https://github.com/micronaut-projects/micronaut-openapi/pull/2321

And with Kotlin DSL I found only this syntax.